ingenialink/virtual/dictionary.py

```diff
@@ -20,13 +19,27 @@ class VirtualDictionary(EthernetDictionary):
 
     """
 
+    def _transform_canopen_index_to_mcb_address(self, index: int, subnode: int) -> int:
+        """CANopen index is an uint16 but MCB address only has 12 bits, so,
```

Comment:
I don't get it. That's why the canopen index is mapped to an MCB address.

Reply:
Unfortunately the doc also refers to axis N= subnode N, we should fix this in the future where it won't hold up.

Reply:
Do you think it's necessary to put it in the docstring? It is an internal formula that converts Index to MCB address, I think it is not necessary to put it in the docstring.